retrieval_service/datastore/helpers.py

```diff
@@ -12,25 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Union
+
 import sqlparse
 
 
-def format_sql(sql: str, params):
+def format_sql(sql: str, params: dict):
```

Comment:
I see you made the changes here -- do you want to just merge this into a single PR?

Reply:
Thanks for the suggestion. Merging this PR into #494.